core/spec/generators/solidus/install/install_generator_spec.rb

```diff
@@ -78,7 +78,7 @@
 
       expect(generator.instance_variable_get(:@run_migrations)).to eq(true)
       expect(generator.instance_variable_get(:@load_seed_data)).to eq(false)
-      expect(generator.instance_variable_get(:@load_sample_data)).to eq(false)
+      expect(generator.instance_variable_get(:@load_sample_data)).to eq(true)
```

Comment:
We can remove the test altogether. `it 'skips sample data if seeds are disabled'` no longer makes sense.